docs/codacy-api/examples/uploading-application-scanning-results.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Instructions on how to upload DAST results to Codacy using the API.
+---
+
+# Uploading DAST results to Codacy
+
+To ensure the security of your web applications, Codacy allows you to upload DAST (dynamic application security testing) results from [Zed Attack Proxy (ZAP)](https://www.zaproxy.org/) directly to Codacy and monitor them as findings under [Security and risk management](../../organizations/managing-security-and-risk.md).
+
+## Uploading app scanning results to Codacy
+
+1.  Use ZAP ([Zed Attack Proxy](https://www.zaproxy.org/)) to perform dynamic app scanning on your application and generate a report in the [Traditional JSON format](https://www.zaproxy.org/docs/desktop/addons/report-generation/report-traditional-json/).
+
+    For details on generating reports in ZAP, refer to the [ZAP documentation](https://www.zaproxy.org/docs/).
+
+1.  Upload the report to the Codacy API endpoint [<span class="skip-vale">uploadDASTReport</span>](https://app.codacy.com/api/api-docs#uploaddastreport):
+
+    <!-- TODO TAROT-2712 is `csrf-token: nocheck` required? -->
+    ```bash
+    curl -X POST https://app.codacy.com/api/v3/organizations/<GIT_PROVIDER>/<ORGANIZATION>/security/tools/dast/<TOOL_NAME>/reports \
+      -H 'api-token: <API_KEY>' \
+      -H 'csrf-token: nocheck' \
```

Comment:
@codacy/tarot Is this mandatory? Looks hacky here

Reply:
I'm trying to figure out if/how we can avoid it -- https://codacy.slack.com/archives/CA8A4L1DY/p1720093167466749

Reply:
We will be able to avoid it as soon as this PR is merged: https://github.com/codacy/codacy-website/pull/1385